fabric/src/test/java/TestRegex.java

```diff
@@ -363,6 +366,42 @@ public void GuildModel_MSG_NEW_OBJECTIVES() {
         p.shouldMatch("§3[INFO]§b New Weekly Guild Objectives are being assigned.");
     }
 
+    @Test
+    public void GuildModel_MSG_TRIBUTE_SCEDULED() {
+        PatternTester p = new PatternTester(GuildModel.class, "MSG_TRIBUTE_SCEDULED");
+        p.shouldMatch("§3[INFO]§b ExamplePlayer1 scheduled 2 Emeralds per hour to Example Guild");
+        p.shouldMatch("§3[INFO]§b ExamplePlayer1 scheduled Ⓙ 1 Crop per hour to Example Guild");
+        p.shouldMatch("§3[INFO]§b ExamplePlayer1 scheduled Ⓚ 1 Fish per hour to Example Guild");
+        p.shouldMatch("§3[INFO]§b ExamplePlayer1 scheduled Ⓒ 1 Wood per hour to Example Guild");
+        p.shouldMatch("§3[INFO]§b ExamplePlayer1 scheduled Ⓑ 1 Ore per hour to Example Guild");
```

Comment:
Does scheduling 2 crop"s" change the message? According to `GuildModel_MSG_TRIBUTE_STOPPED`, it might. If so, please add tests for it.

Reply:
Also, check this with fish/wood too, crops is a special case for plurality.

Reply:
> Does scheduling 2 crop"s" change the message?

Yes, both "Emeralds" and "Crops" have a special case for just a singular resource

> And finally, make sure it can actually be parsed by the GuildProduction (or however) it is called enum, so we don't just match the line, but actually interpret it.

To account for the plural s being missing, the GuildResource is aquired by matching the symbol the actual name of the resource is ignored in this case.

Reply:
Great, thanks! Can you add a final test case to show-case this?